composer.json

```diff
@@ -5,20 +5,20 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.1",
-    "spryker/availability": "^5.0.1 || ^6.0.0",
-    "spryker/cart-extension": "^1.0.0 || ^2.0.0",
+    "spryker/availability": "^5.0.1 || ^6.0.0 || ^8.0.0",
```

Comment:
- Facade method appeared in 6.0.0 only
public function calculateStockForProductWithStore($sku, StoreTransfer $storeTransfer)